docs/workflow/testing/libraries/testing-wasm.md

```diff
@@ -0,0 +1,62 @@
+# Testing Libraries on WebAssembly
+
+In order to be able to run tests, the following JavaScript engines should be installed:
+- V8
+- JavaScriptCore
+- SpiderMonkey
+
+They can be installed as a part of [jsvu](https://github.com/GoogleChromeLabs/jsvu).
+
+Please make sure that a JavaScript engine binary is available via command line,  
+e.g. for V8:
+```bash
+$ v8
+V8 version 8.5.62
+```
+
+If you use `jsvu`, first add its location to PATH variable  
+e.g. for V8
+
+```bash
+PATH=/Users/<your_user>/.jsvu/:$PATH V8
+```
+
+## Building Libs and Tests for WebAssembly
+
+Now we're ready to build everything for WebAssembly:
+```bash
+./build.sh --arch wasm --subset mono -c Release
```

Comment:
Should this also list --arch wasm --os Browser or is it enough with --arch?

Reply:
updated